app/controllers/api/vcs.php

```diff
@@ -556,18 +570,22 @@
         responses: [
             new SDKResponse(
                 code: Response::STATUS_CODE_OK,
-                model: Response::MODEL_DETECTION,
+                model: Response::MODEL_RUNTIME_DETECTION,
+            ),
+            new SDKResponse(
+                code: Response::STATUS_CODE_OK,
+                model: Response::MODEL_FRAMEWORK_DETECTION,
             )
         ]
     ))
     ->param('installationId', '', new Text(256), 'Installation Id')
     ->param('providerRepositoryId', '', new Text(256), 'Repository Id')
     ->param('providerRootDirectory', '', new Text(256, 0), 'Path to Root Directory', true)
+    ->param('type', '', new WhiteList(['runtime', 'framework']), 'Detector type. Must be one of the following: runtime, framework', true)
```

Comment:
```suggestion
    ->param('type', '', new WhiteList(['runtime', 'framework']), 'Detector type. Must be one of the following: runtime, framework')
```
The `true` there means it is optional. I think it should be required, otherwise we don't know what kind of detection they want